apps/meteor/app/api/server/v1/chat.ts

```diff
@@ -74,18 +71,15 @@ API.v1.addRoute(
 				throw new Meteor.Error('error-roomId-param-invalid', 'The "lastUpdate" query parameter must be a valid date.');
 			}
 
-			let result;
-			Meteor.runAsUser(this.userId, () => {
-				result = Meteor.call('messages/get', roomId, { lastUpdate: new Date(lastUpdate) });
-			});
+			const result = await Meteor.call('messages/get', roomId, { lastUpdate: new Date(lastUpdate) });
```

Comment:
for next ones it would be good if you already change to `Meteor.callAsync` , this way we don't need to change it again 😛 

`await Meteor.call()` doesn't have any real impact.. but since we're going to change **all** `Meteor.call()` to `Meteor.callAsync` later on, that's ok 😉